src/pages/_app.jsx

```diff
@@ -102,10 +107,10 @@ const WrappedApp = ({ Component, pageProps }) => {
       }
 
       if (httpError === 401) {
-        return (
-          <NotFoundPage
-            title='Analysis not found'
-            subTitle={'You don\'t have access to this analysis.'}
+        return amplifyConfigured && (
```

Comment:
I think that, after you moved the `!amplifyConfigured ` test to the top, this check is unnecessary.

Reply:
Removing it